JS.md

```diff
@@ -166,6 +166,29 @@ Otherwise, this function performs synchronous compilation of the `BufferSource`:
   the validated `Ast.module`.
 * On failure, a new `WebAssembly.CompileError` is thrown.
 
+### `WebAssembly.Module.exports`
```

Comment:
(race posting)  Hah, three different interpretations of the text.  So would static `Module.imports` methods be preferred even over getters on `Module.prototype` (which is almost equivalent, really...)?